lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -52,6 +53,9 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * @param {@link (RedirectRequest:type)}
      */
     async loginRedirect(request?: RedirectRequest): Promise<void> {
+        if (this.isBrowserEnvironment) {
+            this.broadcastEvent(BroadcastEvent.LOGIN_START, InteractionType.REDIRECT, request);
```

Comment:
And I think that's probably fine, I imagine they will be handled similarily. I think we'll just want to document it well.

Reply:
it could be a pitfall if someone is expecting something to happen on both starts, but as long as the success/error event only fires once I guess it is ok.